src/include/storage/store/nodes_statistics_and_deleted_ids.h

```diff
@@ -179,10 +177,14 @@ class NodesStatisticsAndDeletedIDs : public TablesStatistics {
 
     inline std::unique_ptr<TableStatistics> constructTableStatistic(
         catalog::TableSchema* tableSchema) override {
+        TablePropertyStats propertyStats;
+        for (auto property : tableSchema->getProperties()) {
+            propertyStats[property->getPropertyID()] = std::make_unique<PropertyStatistics>();
+        }
         // We use UINT64_MAX to represent an empty nodeTable which doesn't contain
         // any nodes.
         return std::make_unique<NodeStatisticsAndDeletedIDs>(
-            tableSchema->tableID, UINT64_MAX /* maxNodeOffset */);
+            tableSchema->tableID, UINT64_MAX /* maxNodeOffset */, std::move(propertyStats));
```

Comment:
Instead of moving `propertyStats` into `NodeStatisticsAndDeletedIDs`, you can consider passing the `TableSchema*`, and let the  `NodeStatisticsAndDeletedIDs` to handle the construction of property statistics inside its constructor.